ResearchKit/Common/ORKStepViewController.h

```diff
@@ -350,6 +350,13 @@ ORK_CLASS_AVAILABLE
  */
 - (void)goBackward;
 
+/**
+ Returns a Boolean value indicating whether the step has been presented before.
+ 
+ @return `YES` if the step has been presented; otherwise `NO`.
+ */
+- (BOOL)hasBeenPresented;
```

Comment:
Can you use a property with `readonly` attribute, which is a more prefered way to return a Boolean in RK.